source/developer-guides/ingredients-update.rst

```diff
@@ -0,0 +1,25 @@
+.. _ingredients_upgrade:
+
+Ingredients upgrade
+-------------------
+
+Ingredients such as FSP, VBT and microcode are able to upgrade in |SPN| source code.

```

Comment:
Please provide a little more context - 

Slim Bootloader consumes many silicon specific binary components like the Firmware Support Package binary, the Microcode update, etc. The SBL build process includes steps to pull such binary components from other repositories. The aim of such automatic pulling is to provide an easier out of box developer experience.

However, developers may wish to use different versions of binary components for their specific project needs. The section below provides guidance on how to use different versions of such binary components.

Reply:
Done, thanks for the review.